src/probnum/filtsmooth/gaussfiltsmooth/unscentedkalman.py

```diff
@@ -43,17 +45,18 @@ def __init__(
     ):
         self.disc_model = disc_model
         self.ut = ut.UnscentedTransform(dimension, spread, priorpar, special_scale)
+        super().__init__()
 
     def transition_realization(self, real, start, **kwargs):
         return self.disc_model.transition_realization(real, start, **kwargs)
 
     def transition_rv(self, rv, start, linearise_at=None, **kwargs):
-        if linearise_at is None:
-            sigmapts = self.ut.sigma_points(rv.mean, rv.cov)
-        else:
-            sigmapts = self.ut.sigma_points(linearise_at.mean, linearise_at.cov)
-        proppts = self.ut.propagate(start, sigmapts, self.disc_model.dynamics)
-        meascov = self.disc_model.diffusionmatrix(start)
+        compute_sigmapts_at = linearise_at if linearise_at else rv
```

Comment:
I know that this might even have been my own suggestion, but what do you think about explicitly checking `... if linearise_at is not None else ...`? I'm not sure if that would even matter, the only thing I could think of is if we linearize around some one-dim zero array.

Reply:
I dont think it would matter in the sense that the results are gonna be different. Or is that wrong? Since it is right underneath the signature, it will also be clear what is going on I guess.

Reply:
What I meant here is that if we have `linearize_at = np.array([0])`, then 
```python
compute_sigmapts_at = linearise_at if linearise_at else rv
```
results in `compute_sigmapts_at == rv`, and not `compute_sigmapts_at == linearize_at`, so for this one weird corner case it would actually result in unintended behaviour.

Though of course it's totally unlikely this will ever happen, so I guess it does not really matter^^